tests/backwards_compatibility_tests/test_general_compatibility.py

```diff
@@ -8,7 +8,7 @@
 @pytest.mark.marqo_version('2.11.0') #TODO: Check this again
 class GeneralCompatibilityTest(BaseCompatibilityTestCase):
 
-    image_model = 'open_clip/ViT-B-32/laion2b_s34b_b79k'
+    image_model = 'Marqo/ViT-B-32/laion2b_s34b_b79k'
```

Comment:
I think we should test both not stop testing the old one. Alternatively, I think we can have a unit test with model pair names, and simply assert that other than the name they're exactly identical in the registry.

Reply:
The above works for OpenCLIP models. HF models have a change of type so need to be tested separately